homeassistant/components/bosch_alarm/binary_sensor.py

```diff
@@ -0,0 +1,73 @@
+"""Support for Bosch Alarm Panel binary sensors."""
+
+from __future__ import annotations
+
+from bosch_alarm_mode2 import Panel
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from . import BoschAlarmConfigEntry
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant | None,
```

Comment:
```suggestion
    hass: HomeAssistant,
```
It still can't be none